src/main/resources/META-INF/plugin.xml

```diff
@@ -173,6 +173,9 @@
                 id="lsp.console.explorer.copy.command"
                 class="com.redhat.devtools.lsp4ij.console.explorer.actions.CopyStartServerCommandAction"
                 icon="AllIcons.Actions.Copy"/>
+        <action id="editor.action.triggerSuggest"
```

Comment:
Could you please add this action in a new block actions and add a comment like waiting for standardization of LSP commands and set a link to https://github.com/microsoft/language-server-protocol/issues/788

Reply:
Done